### ISSUE TITLE
[task] Add option to switch between physics controls and navmesh controls

### ISSUE THREAD

**Issue by francy51** [wovencode/OpenMMO-Other#43]
hm we could add a option to the config SO that would allow to switch between "physics" or "navmesh".

The only thing that it would do right now, is to turn off the navmesh validation that happens on login/teleport.

not much but it would help those who want to go for physics instead

**Comment by Xamos** [comment]
Yes, a toggle in the options menu perhaps. You could also have quests (or click destination) that toggle between physics/navmesh on the fly.
